lib/src/lis/protocol.cpp

```diff
@@ -487,6 +487,21 @@ std::string dfs_fmtstr( const dfsr& dfs ) noexcept (false) {
             throw std::runtime_error(fmt::format(msg, size, code, mnem));
         }
 
+        /* Suppressed output
+         *
+         * If the reserved_size in the Spec Block is negative, the bytes is
+         * reserved in the file, but the output is suppressed. I.e. should be
+         * ignored.
+         *
+         * This is communicated in the format-string with the fmt-code
+         * LIS_FMT_SUPPRESS.
+         *
+         */
+        if ( size < 0 ) {
```

Comment:
Add a note somewhere what takes precedence: suppressed channel or samples.

Right now if samples=11, reprcode_size = 4 and suppressed = -2, samples/repcode size will complain. That is as good behavior as any, but explicit note that we are aware of what's going on would be good.